relay-event-schema/src/protocol/span.rs

```diff
@@ -95,6 +102,7 @@ impl From<&Event> for Span {
             received: event.received.clone(),
             start_timestamp: event.start_timestamp.clone(),
             timestamp: event.timestamp.clone(),
+            _metrics_summary: event._metrics_summary.clone(),
```

Comment:
Why does the SDK set the `_metrics_summary` on the event, and not on the span in the first place?

Reply:
Unclear. Maybe @mitsuhiko can shed some light on this.

With this PR, it'll be possible to have `_metrics_summary` on a span. Now, if span metrics extraction is not enabled, we won't persist the span so we won't have the summary stored as well, but we're thinking of indexing all spans (not just the ones we extract metrics for) so that might not be a problem too much longer.

Reply:
It's set on both span and toplevel event because the transaction frustratingly stops being a span in the current protocol.

Reply:
Makes sense.